9.2-jre7/docker-entrypoint.sh

```diff
@@ -94,4 +89,10 @@ if expr "$*" : 'java .*/start\.jar.*$' >/dev/null ; then
 	fi
 fi
 
+if [[ ($1 = java || $1 == */java) && -n "$JAVA_OPTIONS" ]] ; then
+	java="$1"
+	shift
+	set -- "$java" $JAVA_OPTIONS "$@"
```

Comment:
The only problem I can think of moving this to here is that the JAVA_OPTIONs are not applied to the runs of java that occur at line 61 and 80.

Reply:
Can you think of any JVM options that would be relevant to the terminating commands that get executed on that line?

I suppose it wouldn't hurt to just leave the old setting of `JAVA_OPTIONS` in place and to also add this one here.

Reply:
i'm struggling to think of a really valid use-case. perhaps debugging when you might want to pass -verbose.  So not a show stopper.

Reply:
Let's just go with dealing with `$JAVA_OPTIONS` in one place. We can always add it back later if we need to, but I don't think this will break anything (famous last words).

Reply:
ack